### PR TITLE
wip reuse license

### PR DESCRIPTION
**Pull request by fwilhe**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,203 +0,0 @@
-
```
Comment (KevinHudemann) [inline LICENSE:1]: Is it correct to remove this file? I would think it has to stay, ref. todo 3b) in the guide on how to switch to DCO. There it says this file should only be edited, i.e., remove the API and 3rd Party section if it was there.

Comment (fwilhe) [reply LICENSE:1]: yes

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ license agreement. If you do not have a valid license agreement for the use of
+ a particular SAP External Product, then you may only make use of any API Calls
+ in this project for that SAP External Product for your internal, non-productive
+ and non-commercial test and evaluation of such API Calls. Nothing herein grants
+ you any rights to use or access any SAP External Product, or provide any third
+ parties the right to use of access any SAP External Product, through API Calls.
+
+Files: dist/**
+Copyright: ???
+License: ???
```
Comment (misappi) [inline .reuse/dep5:29]: The correct copyright and license must be added.

**Review by sacnl — APPROVED** [review]

**Review by stippi2 — APPROVED** [review]
Looks correct.

**Review by KevinHudemann — APPROVED** [review]

**Review by misappi — APPROVED** [review]

**Pull request merged**